services/housetables/src/main/java/com/linkedin/openhouse/housetables/repository/impl/jdbc/UserTableHtsJdbcRepository.java

```diff
@@ -115,12 +120,13 @@ default void deleteById(UserTableRowPrimaryKey userTableRowPrimaryKey) {
   @Transactional
   @Modifying
   @Query(
-      "UPDATE UserTableRow table SET table.tableId = :toTableId, table.metadataLocation = :metadataLocation, table.databaseId = :toDatabaseId "
+      "UPDATE UserTableRow table SET table.tableId = :toTableId, table.metadataLocation = :metadataLocation, table.databaseId = :toDatabaseId, table.deleted = :deleted "
```

Comment:
This requires adding deleted column in mysql table as well right.

Reply:
Yes